src/bootstrap/compile.rs

```diff
@@ -104,7 +104,11 @@ impl Step for Std {
 
         let out_dir = build.cargo_out(compiler, Mode::Libstd, target);
         build.clear_if_dirty(&out_dir, &builder.rustc(compiler));
-        let mut cargo = builder.cargo(compiler, Mode::Libstd, target, "build");
+        let mut cargo = if compiler.stage == 0 {
+            builder.cargo(compiler, Mode::Libstd, target, "build")
+        }else{
+            builder.cargo(compiler, Mode::Libstd, target, "check")
+        };
```

Comment:
This seems wrong/unrelated? If not, reasoning would be appreciated.

Reply:
Should make this no llvm only. When stage 1 rustc doesnt have llvm support it cant build regular rlibs, but only metadata only rlibs.

Reply:
Ah. I suppose that makes sense.